### ISSUE TITLE
hasMore indication is missing

### ISSUE THREAD

**Issue by rserne** [open-education-api/specification#137]
Each service is returning the pageSize and pageNumber. There is however no indication that a next page exists. An attribute 'hasMore' would be helpful.

**Comment by git4bram** [comment]
At Saxion we implemented a "lastPageNumber" attribute.  A little bit different from the suggestion above, but very helpfull and for probably the same reason.

**Comment by jelmerderonde** [comment]
Suggestions:


```
{
    "pageSize": 1,
    "pageNumber": 1,
    "hasNextPage": true,
    "hasPreviousPage": false,
    "totalPages": 123
    "items": []
}
```
Make`totalPages` optional the rest is mandatory.

**Comment by ruttromp** [comment]
resolved by #169
